.travis.yml

```diff
@@ -33,16 +33,21 @@ script:
 
 jobs:
   include:
-    # Define the commitlint stage
-    - stage: commitlint
+    # Define the `lint` stage (runs `yamlllint` and `commitlint`)
+    - stage: lint
       language: node_js
       node_js: lts/*
       before_install: skip
       script:
+        # Install and run `yamllint`
+        - pip install --user yamllint
+        # yamllint disable-line rule:line-length
+        - yamllint -s . .yamllint pillar.example test/salt/pillar/repositories.pillar.sls test/salt/pillar/preferences.pillar.sls
```

Comment:
Isn't it possible to use some kind of globbing? I think we'll either lose track of the files to lint.

Reply:
Actually, that is picked up automatically via. `ssf-formula`.  The `.` is a glob, where it looks for all of the `.yml` and `.yaml` files recursively.  We supply the rest after it.

Reply:
```jinja
{%- set yamllint_files = ['.', '.yamllint', 'pillar.example'] %}
{#- Don't need to do this in index order but maintaining consistency with `kitchen.yml` #}
{%- for index in range(0, inspec_suites_kitchen | length) %}
{%-   set suite = inspec_suites_kitchen[index] %}
{%-   set pillars_from_files = suite.provisioner.pillars_from_files %}
{%-   for pff in pillars_from_files %}
{%-     for k, v in pff.items() %}
{%-       if v not in yamllint_files %}
{%-         do yamllint_files.append(v) %}
{%-       endif %}
{%-     endfor %}
{%-   endfor %}
{%- endfor %}
```

Reply:
```sls
        # yamllint disable-line rule:line-length
        - yamllint -s {{ yamllint_files | join(' ') }}
```

Reply:
So either we should add a comment, warning that this will be automatically filled in by `ssf-formula` or change that macro to do some kind of (pseudo here):

```yaml
yamllint_files = [ yamllint_files_in_the_formula +
                   ['.', '.yamllint', 'pillar.example']
                 ].flatten.unique
```

right?

Reply:
@javierbertoli Starting with the [source data](https://github.com/myii/ssf-formula/blob/ac88c9e5125a9d7f570d061df67a30b0981bc9a0/ssf/formulas.yaml#L156-L184):

```yaml
    apt:
      context:
        git:
          github:
            repo: apt-formula
        inspec_suites_kitchen:
          0:
            inspec_yml:
              summary: >-
                Verify that the apt repositories are configured correctly
              supports: *supports_debian_ubuntu
            provisioner:
              pillars_from_files:
                - .sls: test/salt/pillar/repositories.pillar.sls
              state_top:
                - '*':
                    - .repositories
                    - .update
          1:
            inspec_yml:
              summary: >-
                Verify that the apt preferences are configured correctly
              supports: *supports_debian_ubuntu
            provisioner:
              pillars_from_files:
                - .sls: test/salt/pillar/preferences.pillar.sls
              state_top:
                - '*':
                    - .preferences
```

You'll see that we're already tracking these "extra" files.  Introducing something like a `yamllint_files_in_the_formula` would be unnecessary duplication.  However, I'm sure the actual implementation itself could be improved.